Dockerfile

```diff
@@ -655,6 +661,10 @@ RUN <<EOT
   docker-proxy --version
 EOT
 
+# used by .devcontainer/devcontainer.json
```

Comment:
Not a blocker; wondering if we should start using the fancy comment-formats to make BuildKit understand it's documenting a stage; https://github.com/moby/buildkit/pull/1693

I'm sure Kevin knows where those surface 🤔